Manual.md

```diff
@@ -0,0 +1,254 @@
+# pgarchive Manual
+
+A PostgreSQL Archive Server providing fast and incremental Backup and Restore Services
+
+
+## Installation
+
+1. You must have PostgreSQL 9.4 installed, including its client and server applications.
+1. You must have `btrfs` installed, from btrfs-progs.
+1. Copy `pgarchive` from this repository to /usr/local/bin or somewhere else in your path. Make sure it has +x permissions.
+1. Edit `pgarchive's` `export PATH=` line near the top so it can find the right (9.4) PostgreSQL applications and `btrfs`.
+
+If you want fancy bash completion to work you should execute the following, or better yet put it in a .bashrc or equivalent file:
+
+```
+eval "$(pgarchive bash-completion)"
+```
+
+All invocations of pgarchive require $PGARCHIVE to be the path to a backup container, which must be located on a btrfs filesystem. You might want to add something like this to your .bashrc too:
+
+```
+export PGARCHIVE=/path/to/container
+```
+
+For high-load DBs at emarsys.com the following btrfs mount options work best. You do need `user_subvol_rm_allowed` for snapshot expiry to work, the rest merely improve performance for us. See [btrfs mount options] for more details.
+
+```
+user_subvol_rm_allowed,noatime,nobarrier,enospc_debug,nodatacow,metadata_ratio=6
+```
+
+Now you need to use `pgarchive container init`, possibly with more configuration, to create container(s). Re-starting containers on server re-boots is left up to you. (TODO: add sysV and systemd service scripts to repository)
```

Comment:
In the next section you are referring to the word container. Maybe you can spend some words in order to explain what a container is in this context.